### ISSUE TITLE
Pass Fail status should be indicated with a character for accessibility

### ISSUE THREAD

**Issue by AMWatty** [getgauge/Flash#2]
Currently, the flash report indicates pass/fail status with text color.  Red-Green colorblind people might have trouble distinguishing the results unless a character is added indicating pass fail.
For example: gauge-plugin for VSCode uses an X character for failures:  ✘
for pass results it a check mark:  ✔
